src/dotnet/commands/dotnet-compile/Program.cs

```diff
@@ -71,16 +71,35 @@ private static bool OnExecute(List<ProjectContext> contexts, CompilerCommandApp
 
             var managedOutput = outputPaths.CompilationFiles.Assembly;
 
+            // Create the library exporter
+            var exporter = context.CreateExporter(args.ConfigValue);
```

Comment:
Don't understand how this works. Don't you need to at least use the correct runtime id? Do the core clr implementations and CoreRT implementations line up?